Libplanet/Net/Swarm.cs

```diff
@@ -120,7 +120,7 @@ static Swarm()
             IEnumerable<IceServer> iceServers = null,
             DifferentAppProtocolVersionEncountered differentAppProtocolVersionEncountered = null,
             IEnumerable<PublicKey> trustedAppProtocolVersionSigners = null,
-            SwarmOptions swarmOptions = null)
+            SwarmOptions options = null)
```

Comment:
I guess we may need to describe this change to CHANGES.md.

Reply:
I didn't add it to the changelog because the constructor is internal.

Reply:
🆗